### PR TITLE
Update docs for cluster up

### PR DESCRIPTION
**Pull request by SoManyHs**

### CODE REVIEW

**Comment by SoManyHs** [comment]
Also adds warning for irrelevant capability-iam flag:
```
~/go/src/github.com/aws/amazon-ecs-cli(update-cluster-up-docs)$ ecs-cli up -e --cluster test-empty --capability-iamWARN[0000] The '--capability-iam' flag will be ignored when creating an empty cluster 
INFO[0000] Created cluster                               cluster=test-empty region=us-west-2
Cluster creation succeeded.

```

File: README.md [hunk 1]
```diff
@@ -180,53 +180,13 @@ service up`, regardless of which launch type is configured for your cluster (see
...
-                                     This option is useful for cleaning up stale resources from previous failed attempts.
-   --launch-type value               [Optional] Specifies the launch type. Options: EC2 or FARGATE. Overrides the default launch type stored in your cluster configuration.
-                                     Defaults to EC2 if a cluster configuration is not used.
-   --region value, -r value          [Optional] Specifies the AWS region to use. Defaults to the region configured using the configure command
-   --cluster-config value            [Optional] Specifies the name of the ECS cluster configuration to use. Defaults to the default cluster configuration.
-   --ecs-profile value               [Optional] Specifies the name of the ECS profile configuration to use. Defaults to the default profile configuration. [$ECS_PROFILE]
-   --aws-profile value               [Optional]  Use the AWS credentials from an existing named profile in ~/.aws/credentials. [$AWS_PROFILE]
-   --cluster value, -c value         [Optional] Specifies the ECS cluster name to use. Defaults to the cluster configured using the configure command
+ecs-cli up
```
Comment (allisaurus) [inline README.md]: should this be wrapped with in-line code ticks, a la `ecs-cli up` ?

Comment (SoManyHs) [reply README.md]: Having the three tickets before and after make it a code block. You can view the rendered version [here] (https://github.com/SoManyHs/amazon-ecs-cli/blob/c33dc458cb4a3e3913b628f7c06577751b869b7d/README.md#creating-an-ecs-cluster)

Comment (allisaurus) [reply README.md]: yeah totally missed those! ignore me

**Pull request merged**